components/Caretaker.js

```diff
@@ -50,10 +49,10 @@ const Caretaker = ({history, navigation}) => {
           keyExtractor={(item) => item.id.toString()}
         />  
 
-      <BottomSheet isVisible={isVisible} containerStyle={{}}>
-        <Button buttonStyle={styles.buttonStyle} title='x' titleStyle={styles.titleStyle} onPress={() => setIsVisible(false)}></Button>
-        <AddTimer addTimer={addTimer} setIsVisible={setIsVisible}/>
-      </BottomSheet>
+      <Modal visible={modalVisible} animationType="slide">
+        <Button buttonStyle={styles.buttonStyle} title='Close Menu' titleStyle={styles.titleStyle} onPress={() => setIsVisible(!modalVisible)}></Button>
+        <AddTask addTimer={addTimer} setIsVisible={setIsVisible}/>
```

Comment:
Also for semantics, we could probably change setIsVisible to setModalIsVisible, but honestly these are just small nit-picky things, that don't "need" to be changed. Mainly just to add references.